README.md

```diff
@@ -44,12 +44,15 @@ See [here](./docs/iam-policy.md) for required IAM policies.
 
 ## Components
 
-  There are 2 components:
+  There are 3 components:
 
   * [CNI Plugin](https://kubernetes.io/docs/concepts/cluster-administration/network-plugins/#cni), which will wire up the host's and pod's network stack when called.
   * `ipamd`, a long-running node-Local IP Address Management (IPAM) daemon, is responsible for:
     * maintaining a warm-pool of available IP addresses, and
     * assigning an IP address to a Pod.
+  * [Network Policy Controller](https://github.com/aws/amazon-network-policy-controller-k8s), an optional controller used to implement network policies. 
```

Comment:
I am going to add all of these, just because I have a hard time keeping track of them 😂

    * Existing VPC CNI - https://github.com/aws/amazon-vpc-cni-k8s
    * New - Node Agent - https://github.com/aws/aws-network-policy-agent
    * New - Controller - https://github.com/aws/amazon-network-policy-controller-k8s
    * New - EBPF SDK  - https://github.com/aws/aws-ebpf-sdk-go

Reply:
I think we should skip those, as they are not components in this repo or components built by this repo

Reply:
how about something like "related components"?

Reply:
Great idea, that's perfect!